ansible/roles/schulcloud-server-core/tasks/main.yml

```
@@ -269,6 +269,26 @@
       namespace: "{{ NAMESPACE }}"
       template: tldraw-svc-monitor.yml.j2
     when: WITH_TLDRAW is defined and WITH_TLDRAW|bool
+  
+  - name: course export import deployment
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: course-export-import-deployment.yml.j2
```

No Deployment for it if not needed
```suggestion
  - name: course export import deployment
    kubernetes.core.k8s:
      kubeconfig: ~/.kube/config
      namespace: "{{ NAMESPACE }}"
      template: course-export-import-deployment.yml.j2
    when: WITH_COURSE_EXPORTER is defined and WITH_COURSE_EXPORTER|bool
```

---

done ✅